docs/source/api/references/index.rst

```diff
@@ -5,6 +5,8 @@ API Reference
 
 The released APIs include the necessary components that can build GraphStorm models, training and inference pipelines. For examples about how to use these APIs, please refer to the :ref:`API Programming Examples<programming-examples>`.
 
+.. note:: This API reference does not include the components for multi-task learning, which is the new yet experimental feature of Graphstorm v0.3.
```

Comment:
```suggestion
.. note:: This API reference does not include the components for multi-task learning, which is still an experimental feature of Graphstorm v0.3.
```

Could also say we plan to add those docs?

Reply:
We may not make promise. I change to add a reason.